web/client/components/geostory/layouts/sections/__tests__/Banner-test.jsx

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import expect from 'expect';
+import { testToolbarButtons } from './testUtils';
+import Banner, { DEFAULT_BANNER_HEIGHT } from '../Banner';
+
+describe('Banner component', () => {
+    beforeEach((done) => {
+        document.body.innerHTML = '<div id="container"></div>';
+        setTimeout(done);
+    });
+    afterEach((done) => {
+        ReactDOM.unmountComponentAtNode(document.getElementById("container"));
+        document.body.innerHTML = '';
+        setTimeout(done);
+    });
+    it('Banner rendering with defaults', () => {
+        ReactDOM.render(<Banner />, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+    });
+    it('Banner background rendering (image)', () => {
+        const IMAGE_SRC = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8BQDwAEhQGAhKmMIQAAAABJRU5ErkJggg==';
+        const CONTENTS = [
+            {
+                id: '000',
+                type: 'column',
+                background: {
+                    type: 'image',
+                    src: IMAGE_SRC,
+                    lazy: false
+                },
+                html: '<h1>Banner</h1>'
+            }
+        ];
+        ReactDOM.render(<Banner contents={CONTENTS} mode="edit"/>, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+        const img = el.querySelector('img');
+        expect(img).toExist();
+        expect(img.getAttribute('src')).toBe(IMAGE_SRC);
+        const contentToolbar = container.querySelector('.ms-content-toolbar');
+        expect(contentToolbar).toExist();
+
+        const buttonsInToolbar = container.querySelectorAll('.ms-section-background-container .btn-group .glyphicon');
+        expect(buttonsInToolbar).toExist();
+        expect(buttonsInToolbar.length).toBe(7);
+        testToolbarButtons(["pencil", "height-view", "fit-contain", "size-extra-large", "align-center", "dropper", "trash"], container);
+
+    });
+    it('Banner rendering cover set to true', () => {
+        const VIEW_HEIGHT = 500;
+        const CONTENTS = [
+            {
+                id: '000',
+                type: 'text',
+                html: '<h1>Banner</h1>'
```

Comment:
Please remove `html: '<h1>Banner</h1>'` and `type: text` because they are not used by banner